data/json/items/tool_armor.json

```diff
@@ -753,6 +685,840 @@
         "coverage" : 75,
         "material_thickness" : 1
     },
+    {
+        "id": "mask_filter",
+        "type": "TOOL_ARMOR",
+        "name": "filter mask",
+        "description": "A mask that straps over your mouth and nose and filters air.  Protects from smoke, dust, and other contaminants quite well.  It is open and must be activated before use.",
+        "weight": 260,
+        "volume": 2,
+        "price": 3500,
+        "to_hit": 1,
+        "material": [ "plastic" ],
+        "symbol": "[",
+        "color": "light_gray",
+        "covers": [ "MOUTH" ],
+        "coverage": 100,
+        "encumbrance": 20,
+        "warmth": 10,
+        "material_thickness": 2,
+        "environmental_protection": 1,
+        "max_charges": 100,
+        "initial_charges": 100,
+        "charges_per_use": 1,
+        "ammo": "gasfilter_s",
+        "use_action": {
+            "type": "transform",
+            "msg": "You activate your %s.",
+            "target": "mask_filter_on",
+            "active": true,
+            "need_charges": 1,
+            "need_charges_msg": "The %s's filter is used up."
+        }
+    },
+    {
+        "id": "mask_filter_on",
+        "type": "TOOL_ARMOR",
+        "name": "filter mask (on)",
+        "description": "A mask that straps over your mouth and nose and filters air.  Protects from smoke, dust, and other contaminants quite well.",
+        "weight": 260,
+        "volume": 2,
+        "price": 3500,
+        "to_hit": 1,
+        "material": [ "plastic" ],
+        "symbol": "[",
+        "color": "light_gray",
+        "covers": [ "MOUTH" ],
+        "coverage": 100,
+        "encumbrance": 20,
+        "warmth": 10,
+        "material_thickness": 2,
+        "environmental_protection": 7,
+        "max_charges": 100,
+        "turns_per_charge": 100,
+        "ammo": "gasfilter_s",
+        "revert_to": "mask_filter",
+        "use_action": { "type": "transform", "menu_text": "Turn off", "msg": "Your %s deactivates.", "target": "mask_filter" }
```

Comment:
What does this `"Your %s deactivates."` mean? How it is possible to deactivate filter mask?

Reply:
Another copypaste from rebreather mask, Chemical rebreather cartridge must be a single use item, simular to glowstick, but now it can be deactiveted.